R/spatial_block_cv.R

```diff
@@ -98,21 +103,35 @@ spatial_block_cv <- function(data,
     grid_box[3] <- grid_box[3] + abs(grid_box[3] * 0.001)
     grid_box[4] <- grid_box[4] + abs(grid_box[4] * 0.001)
   }
+
+  if (missing(radius)) radius <- NULL
+  if (missing(buffer)) buffer <- NULL
+
   grid_blocks <- sf::st_make_grid(grid_box, ...)
   split_objs <- switch(
     method,
-    "random" = random_block_cv(data, grid_blocks, v),
-    systematic_block_cv(data, grid_blocks, v, ordering = method, relevant_only)
+    "random" = random_block_cv(
+      data,
+      grid_blocks,
+      v,
+      radius,
+      buffer
+    ),
+    systematic_block_cv(
+      data,
+      grid_blocks,
+      v,
+      ordering = method,
+      relevant_only,
+      radius,
+      buffer
+    )
   )
   v <- split_objs$v[[1]]
   split_objs$v <- NULL
 
-  ## We remove the holdout indices since it will save space and we can
-  ## derive them later when they are needed.
-  split_objs$splits <- map(split_objs$splits, rm_out)
-
   ## Save some overall information
-  cv_att <- list(v = v)
+  cv_att <- list(v = v, radius = radius, buffer = buffer)
```

Comment:
`radius` and `buffer` are added in order for `autoplot` to, at some point down the line, draw the inclusion and exclusion polygons that were used for each fold.